packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { ReactNode } from 'react';
 import CTraderIcon from '../../public/images/cfd/ctrader.svg';
 import DerivXIcon from '../../public/images/cfd/derivx.svg';
+import FinancialEUMT5Icon from '../../public/images/cfd/eu-mt5-financial.svg';
```

Comment:
@thisyahlen-deriv I believe we have this icon in quill/icons, can you check?

Reply:
I have another PR working on replacing the icons from quill.

Reply:
Still figuring out the /dist thing nilofaar mentioned